src/nomad_simulations/schema_packages/model_system.py

```diff
@@ -979,7 +982,27 @@ class ModelSystem(System):
     particle_states = SubSection(
         section_def=ParticleState.m_def,
         repeats=True,
-        description='Particle states',
+        description="""
+            Per-particle state information.
+
+            - Ordering: In the representative (top-level) `ModelSystem` this list
+            has exactly `n_particles` entries and the order matches
+            `positions[i]`.
+
+            - Normalisation behavior: If parsers fill the list only in child
```

Comment:
you could explain the normalisation behavior in the normalizer function :)

Reply:
Would rephrase it without mentioning parsing necessarily:

```suggestion
            - Normalisation behavior: construct a common root from the subsystems.
```